frontend/src/router/routeHandlers.ts

```diff
@@ -29,15 +31,25 @@ import type { RouteLocationNormalized } from 'vue-router';
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
+    let applicationAdmins;
+    let userRolesFetchResult;
     // Requires fetching applications the user administers.
     await asyncWrap(fetchApplications());
-    const userRolesFetchResult = await asyncWrap(
-        fetchUserRoleAssignments(selectedApplication.value?.application_id)
-    );
-    Object.assign(to.meta, { userRoleAssignments: userRolesFetchResult.data });
+    if(selectedApplication.value?.application_id === FAM_APPLICATION_ID) {
```

Comment:
yeah, like if we created a selectedApplicationId variable in the state, we can use it in all places I mentioned above, instead of the short name, thanks!!